tensilelite/rocisa/CMakeLists.txt

```diff
@@ -26,6 +26,17 @@ set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Make sure the folder is the same as the one CXX_COMPILER is in if ROCM_PATH is not set
+if(NOT DEFINED ROCM_PATH)
```

Comment:
This should not be needed and we can't be doing this kind of path manipulation assuming a fixed rocm_path.

Still reading the rest to figure out why this is needed.

Reply:
I was told that we have to use env ROCM_PATH to get the root folder. Is there a new way to get the root folder?

Reply:
There is no root folder. I'm curious why this patch needs it.

Reply:
the root of the rocm folder.

Reply:
I guess we have to go the CMAKE_PREFIX_PATH, but I'm not sure if the CI team is aware of this.

Reply:
Let me look more closely at it tomorrow. We are already locating hip and should find this the same way (they are distributed together).

The other part that I wasn't sure if you were trying to do is to make sure that the c++ compiler, which is still used for much of the project, is from the same distribution as comgr. It would be gross if those two got out of alignment, but there isn't a good was to ensure that.